### PR TITLE
feat(test-helper-appid): add test helper for creating app id test users

### LINKED ISSUE #230: As an engineer, I can create test user JWT tokens

### PR DESCRIPTION
**Pull request by ianwremmel**
Fixes #230

### CODE REVIEW

**Comment by nickclar** [comment]
Why did you choose to call it appid instead of app-id or appID?

File: packages/test-helper-appid/src/router.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+ *
+ * Copyright (c) 2015-2016 Cisco Systems, Inc. See LICENSE file.
+ * @private
+ */
+
+'use strict';
+
+var bodyParser = require('body-parser');
+var express = require('express');
+var createUser = require('./create-user');
```
Comment (nickclar) [inline packages/test-helper-appid/src/router.js]: do you care about alphabetical order in these vars

File: packages/test-helper-appid/src/router.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+// Enable JSON bodies
+// ------------------
+
+router.use(bodyParser.json({
+  strict: false
+}));
+
+router.post('/', function(req, res, next) {
+  if (!req.body.subject) {
```
Comment (nickclar) [inline packages/test-helper-appid/src/router.js]: check for req.body unless a known guaranteed value

**Comment by ianwremmel** [comment]
exhaustion? :)

appID wouldn't fly because of the caps. I could probably switch to app-id.

File: packages/test-helper-appid/src/router.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+var bodyParser = require('body-parser');
+var express = require('express');
+var createUser = require('./create-user');
+
+/* eslint new-cap: [0] */
+var router = module.exports = express.Router();
+
+// Enable JSON bodies
+// ------------------
```
Comment (nickclar) [inline packages/test-helper-appid/src/router.js]: is this line just a separator or does it have meaning in es6?

File: packages/test-helper-appid/package.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@ciscospark/test-helper-appid",
+  "version": "0.7.0-alpha",
+  "description": "",
```
Comment (nickclar) [inline packages/test-helper-appid/package.json]: no description?

File: packages/test-helper-appid/src/router.js (hunk 3 above)
Comment (ianwremmel) [inline packages/test-helper-appid/src/router.js]: just a separator. when all of the test-helper-server code was in a single file, i used comments like this to break up sections. that style sort of continued when I broke it up into multiple files, so I kept it here.

File: packages/test-helper-appid/src/router.js (hunk 2 above)
Comment (ianwremmel) [inline packages/test-helper-appid/src/router.js]: I believe it's guaranteed if the request has the json content type.

Comment (nickclar) [inline packages/test-helper-appid/src/router.js]: ok

**Pull request merged**